fiatconnect-api.md

```diff
@@ -2439,7 +2504,50 @@ If `keyType` is `PHONE`, `key` MUST be an 11-digit [Brazilian mobile phone numbe
 
 Otherwise, if `keyType` is `RANDOM`, `key` MUST be [UUID](https://en.wikipedia.org/wiki/Universally_unique_identifier). If `keyType` is `RANDOM`, `key` represents a random key generated by the user's bank.
 
+### 9.3.3. User Action Details Schemas
+
+All User Action Details Schemas supported by FiatConnect MUST contain the `userActionType` field. The `userActionType` field denotes which `TransferInUserActionDetailsEnum` this schema is associated with; its value MUST be one of the values in `TransferInUserAction`, and it MUST be unique across all supported User Action Details Schemas.
+
+#### 9.3.3.1. `PIXUserAction`
+
+`PIXUserAction` is a User Action Details Schema for transfers in requiring user action which use the `PIXAccount` Fiat Account Schema.
+
+```
+{
+	userActionType: `TransferInUserActionDetailsEnum.PIXUserAction`,
+	pixString: `string`
+}
+```
+
+The `pixString` field is a copy-and-pastable code that the user can enter into the Pix payment system to initiate the transfer of fiat funds to the provider.
+
+#### 9.3.3.2. `IBANUserAction`
+
+`IBANUserAction` is a User Action Details Schema for transfers in requiring user action which use the `IBANAccount` Fiat Account Schema.
+
+```
+{
+	userActionType: `TransferInUserActionDetailsEnum.IBANUserAction`,
+	iban: `string`,
+	bic: `string`
+}
+```
+
+The `iban` field represents the IBAN number for the provider-controlled bank account that the user should send funds to. The `bic` field represents the
+[Bank Identifier Code](https://www.business.hsbc.uk/en-gb/solutions/iban-and-bic) associated with the institution with which the provider-controlled bank account is registered.
+
+#### 9.3.3.3. `PSEUserAction`
+
+`PSEUserAction` is a User Action Details Schema for transfers in requiring use of the Colombian [PSE payment system](https://www.pse.com.co/persona).
+
+```
+{
+	userActionType: `TransferInUserActionDetailsEnum.PSEUserAction`,
+	url: `string`,
+}
+```
```

Comment:
Should probably add one more here for FSP (Faster Payment) a UK payment scheme where IBAN is not used but rather Account Number and Sort code, the Schema could look like this:
```{
	userActionType: `TransferInUserActionDetailsEnum.IBANUserAction`,
	accountNumber: `string`,
	sortCode: `string`
}```

This is described here on the governments website: https://design-system.service.gov.uk/patterns/bank-details/

Reply:
Will you require FSP for user-initiated cash-ins? If you'll require this after this PR is merged, I'm fine with including this here, but if this is more speculatively useful, I'd rather not include it until someone definitely has a need for it.